lib/app_profiler/yarn/command.rb

```diff
@@ -5,15 +5,20 @@ module Yarn
     module Command
       class YarnError < StandardError; end
 
+      GECKO_VIEWER_PACKAGE = AppProfiler.gecko_viewer_package
```

Comment:
Assigning a configurable value to a constant doesn't really make sense to me. We should just inline the string here instead.